scm/driver/harness/webhook.go

```diff
@@ -271,6 +275,7 @@ func convertPullReq(pr pullReq, ref ref, commit hookCommit) scm.PullRequest {
 		Closed: pr.State != "open",
 		Source: pr.SourceBranch,
 		Target: pr.TargetBranch,
+		Merged: pr.State == "merged",
 		Fork:   "fork",
```

Comment:
should we keep this empty? not sure why this says "fork"

Reply:
don't want to change something which is working in this PR ⛔